module3/intermission/index.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Intermission Work
+---
+
+Before staring Module 3, spend a minimum of 5 hours working through the following activities.
+
+### Required
+[Codeacademy Learn HTML - Elements and Structure](https://www.codecademy.com/learn/learn-html)  
```

Comment:
Agree with making this required! Trying to think what we could add as a deliverable here for accountability. Maybe they send us before day 1 the furthest exercise they got to in HTML and CSS and what their favorite exercise was?

Reply:
Yea... maybe I'll go through the first section today, and try to find a good screenshot or something that shows that they have completed this 🧐